syntax/org.vim

```diff
@@ -111,5 +111,22 @@ highlight def link orgHyperCentre Comment
 highlight def link orgHyperLeft Comment
 highlight def link orgHyperRight Comment
 
+" TeX
+" Support for both inline and block based embedded latex
+" eg: $Latex$ for inline or $$ LaTeX $$ for a block
+" Note:
+" - $LaTeX$ uses the tex.vim syntax for its conceal properties
+" - Inspired by https://github.com/vim-pandoc/vim-pandoc-syntax
+" - the conceal settings follows your g:tex_conceal setting for
+"   more info run :h tex-conceal.
+" Ref: https://orgmode.org/manual/LaTeX-fragments.html#LaTeX-fragments
+syntax include @LATEX syntax/tex.vim
+syntax region orgMath     start="\\begin[.*]{.*}"  end="\\end{.*}" 		 keepend contains=@LATEX
+syntax region orgMath     start="\\begin{.*}" 	 end="\\end{.*}" 		 keepend contains=@LATEX
+syntax region orgMath     start="\\\[" 				 end="\\\]" 			 keepend contains=@LATEX
+syntax region orgMath     start="\S\@<=\$\|\$\S\@="   end="\S\@<=\$\|\$\S\@="  keepend oneline contains=@LATEX
+syntax region orgMath     start=/\$\$/                end=/\$\$/              keepend contains=@LATEX
+syntax region orgMath     start=/\\\@<!\\\[/          end=/\\\@<!\\\]/        keepend contains=@LATEX
```

Comment:
I'm not really sure what the difference between this line and line 126 is.

Reply:
I originally made this for vim-dotoo and attempted to add escaping but i had issues getting it to work. Anyways i must have forgotten to remove it when I started creatign this pullrequest. Sorry about that.